ssz/sedes/byte_vector.py

```diff
@@ -0,0 +1,49 @@
+from typing import (
+    Union,
+)
+
+from ssz.exceptions import (
+    SerializationError,
+)
+from ssz.sedes.vector import (
+    CompositeSedes,
```

Comment:
There are `ssz.sedes.base.CompositeSedes` and `ssz.sedes.vector.Vector`, I think you wanted `ssz.sedes.base.CompositeSedes`?